src/project/project-actions.tsx

```diff
@@ -541,6 +565,34 @@ export class ProjectActions {
     }
   };
 
+  isDefaultProjectName = (): boolean =>
+    this.project.name === this.intl.formatMessage({ id: "untitled-project" });
```

Comment:
Another option would be checking whether `this.project.fs.name` was defined. I guess they might differ if you "OK" the project name dialog but stopping nagging you if you do that might be a good thing.

Reply:
I think that is a better behaviour, [`2cf98b7`](https://github.com/microbit-foundation/python-editor-next/pull/763/commits/2cf98b77f736e0600c02f978367491783250b2f4).